lox_services/persistence/storage/storage.py

```diff
@@ -2,14 +2,15 @@
 import os
 import shutil
 import urllib
-from typing import Union
+from typing import Union, List
 
 from google.api_core.page_iterator import HTTPIterator
 from google.api_core import exceptions
 from google.cloud.storage import Client, Blob
 
 
 from lox_services.persistence.config import SERVICE_ACCOUNT_PATH
+from lox_services.persistence.storage import storage
```

Comment:
?

Reply:
Oops, imported to test on a separate file
Removed it ✅